libs/s25main/ingameWindows/iwSettings.cpp

```diff
@@ -15,52 +15,64 @@
     : IngameWindow(CGI_SETTINGS, IngameWindow::posLastOrCenter, Extent(370, 172), _("Settings"),
                    LOADER.GetImageN("resource", 41))
 {
-    AddText(46, DrawPoint(15, 40), _("Fullscreen resolution:"), COLOR_YELLOW, FontStyle{}, NormalFont);
-    AddText(47, DrawPoint(15, 85), _("Mode:"), COLOR_YELLOW, FontStyle{}, NormalFont);
-    AddCheckBox(4, DrawPoint(200, 124), Extent(150, 26), TextureColor::Grey, _("Statistics Scale"), NormalFont, false);
-    GetCtrl<ctrlCheck>(4)->setChecked(SETTINGS.ingame.scale_statistics);
-
-    // "Vollbild"
-    ctrlOptionGroup* optiongroup = AddOptionGroup(3, GroupSelectType::Check);
-    optiongroup->AddTextButton(1, DrawPoint(200, 70), Extent(150, 22), TextureColor::Grey, _("Fullscreen"), NormalFont);
-    optiongroup->AddTextButton(2, DrawPoint(200, 95), Extent(150, 22), TextureColor::Grey, _("Windowed"), NormalFont);
+    AddText(ID_txtResolution, DrawPoint(15, 40), _("Fullscreen resolution:"), COLOR_YELLOW, FontStyle{}, NormalFont);
+    auto* cbResolution =
+      AddComboBox(ID_cbResolution, DrawPoint(200, 35), Extent(150, 22), TextureColor::Grey, NormalFont, 110);
 
-    // "Vollbild" setzen
-    optiongroup = GetCtrl<ctrlOptionGroup>(3);
-    optiongroup->SetSelection((SETTINGS.video.fullscreen ? 1 : 2)); //-V807
     VIDEODRIVER.ListVideoModes(video_modes);
-
-    // "Auflösung"
-    AddComboBox(0, DrawPoint(200, 35), Extent(150, 22), TextureColor::Grey, NormalFont, 110);
-
-    // Und zu der Combobox hinzufügen
     for(unsigned i = 0; i < video_modes.size(); ++i)
     {
         // >=800x600, alles andere macht keinen Sinn
         if(video_modes[i].width >= 800 && video_modes[i].height >= 600)
         {
-            GetCtrl<ctrlComboBox>(0)->AddString(helpers::format("%ux%u", video_modes[i].width, video_modes[i].height));
-
-            // Ist das die aktuelle Auflösung? Dann selektieren
+            cbResolution->AddString(helpers::format("%ux%u", video_modes[i].width, video_modes[i].height));
             if(video_modes[i] == SETTINGS.video.fullscreenSize)
-                GetCtrl<ctrlComboBox>(0)->SetSelection(i);
+                cbResolution->SetSelection(i);
         } else
         {
             video_modes.erase(video_modes.begin() + i);
             --i;
         }
     }
+    AddText(ID_txtFullScreen, DrawPoint(15, 85), _("Mode:"), COLOR_YELLOW, FontStyle{}, NormalFont);
+    ctrlOptionGroup* optiongroup = AddOptionGroup(ID_grpFullscreen, GroupSelectType::Check);
+    optiongroup->AddTextButton(ID_btOn, DrawPoint(200, 70), Extent(150, 22), TextureColor::Grey, _("Fullscreen"),
+                               NormalFont);
+    optiongroup->AddTextButton(ID_btOff, DrawPoint(200, 95), Extent(150, 22), TextureColor::Grey, _("Windowed"),
+                               NormalFont);
+    optiongroup->SetSelection(SETTINGS.video.fullscreen); //-V807
+
+    AddCheckBox(ID_cbInvertMouse, DrawPoint(15, 124), Extent(150, 26), TextureColor::Grey, _("Invert mouse"),
```

Comment:
Would change it here as well. Once again title case, same as "Statistics Scale" below.

```suggestion
    AddCheckBox(ID_cbInvertMouse, DrawPoint(15, 124), Extent(150, 26), TextureColor::Grey, _("Invert Mouse Pan"),
```